### PR TITLE
Jira bug

### PR DESCRIPTION
**Pull request by KseniaLL**
User cannot be completely removed from jira. It was creating issues. Now if user already exists, it still can be added to the group

### CODE REVIEW

**Review by dnorth98 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -102,6 +102,14 @@ A sample config file template is provided in /project/samples
 1. Enter a username and password of your Jira administrator account
 2. Provide groups to add the new user to
 
+### Google Spreadsheet
```
Comment (haggaret) [inline README.md:105]: I'm guessing this is not related in any way to the code changes below? It just confused me that there was something mentioning Google Spreadsheet in the README and then no code related to it in the code change. :)

**Review by haggaret — COMMENTED** [review]
Looks good.
